rumqttc/src/lib.rs

```diff
@@ -404,6 +409,11 @@ impl NetworkOptions {
         self.conn_timeout
     }
 
+    pub fn set_bind_addr(&mut self, bind_addr: SocketAddr) -> &mut Self {
```

Comment:
What about using `impl` [std::net::ToSocketAddrs](https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html#)?

Reply:
> What about using `impl` [std::net::ToSocketAddrs](https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html#)?

Like [this](https://github.com/cavivie/rumqtt/commit/5d7ec634edbfccbe5929ae7b72adc7165c7f98fd#diff-a7c14b22199b25375255ee5a3d137a7f77ad9fdde94ff211cd27c6441f3cbd1dR412-R419)?

Reply:
Cool, to_socket_addrs() is very useful.